### PR TITLE
test: fix && update flaky snapshots

### PR DESCRIPTION
**Pull request by mattlewis92**
Notable changes:
* removed the webpack manifest from snapshots to remove noise
* wrapped the loader with a hack to force the resourcePath to be consistent across builds so the instrumented output is always the same

### CODE REVIEW

File: test/webpack.js [hunk 1]
```diff
@@ -19,6 +19,11 @@ export default function ({ fixture = 'basic.js', options, extend = {} } = {}) {
       }],
     },
     ...extend,
+    plugins: [
+      new webpack.optimize.CommonsChunkPlugin({
```
Comment (michael-ciniawsky) [inline test/webpack.js]: Does this split the webpack runtime from the test bundle ?

File: test/wrap-loader.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import loader from '../src/cjs';
```
Comment (michael-ciniawsky) [inline test/wrap-loader.js]: 👍  Put `wrap-loader.js` && `webpack.js` into a e.g `utils` folder and rename `wrap-loader => loader.js` please

```
|– test
| |– __snapshots__
| |– index.test.js
| |– ${option.name}.test.js
| |– ...
| |– utils
| | |– webpack.js (Compiler/Runner)
| | |– stats.js (Generic Stats/Compilation Normalizer [ TODO ])
| | |– loader.js (Generic Loader Normalizer)
``` 

Sry to bother you with this all the time, it's drafting out the base test system on the job :D

File: test/webpack.js (hunk 1 above)
Comment (mattlewis92) [inline test/webpack.js]: Yup, you can see the change in the [snapshot diff](https://github.com/webpack-contrib/istanbul-instrumenter-loader/pull/66/files#diff-26e4d9ebc388064364a6cf68d73f91f8), its much cleaner 😄

**Comment by mattlewis92** [comment]
@michael-ciniawsky done!

**Comment by d3viant0ne** [comment]
Ship it @michael-ciniawsky

**Pull request merged**